sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe it was a mistake, you said:

> You should rename it to sale_order (**model name**, not module name).

The model name is `SaleOrderLine`, correct.

Indeed the **module** should be `sale_order.py`.